python/.flake8

```diff
@@ -0,0 +1,7 @@
+[flake8]
```

Comment:
does it make sense to run linters for zero lines of foxglove-written python?

Reply:
A fair question. I guess this sets us up for it if we ever want to add code here. But I suppose we’d likely add any supporting code in a separate package, not this package.